store/operations_test.go

```diff
@@ -0,0 +1,88 @@
+package store
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/go-amino"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	db "github.com/tendermint/tm-db"
+	"testing"
+	"time"
+)
+
+var (
+	testSKName = "test"
+	testKey    = []byte("testkey")
+)
+
+func upsert(ctx sdk.Context, sk sdk.StoreKey, cdc *amino.Codec, key []byte, val interface{}) {
+	s := ctx.KVStore(sk)
+	s.Set(key, cdc.MustMarshalBinaryBare(val))
+}
+
```

Comment:
All tests should make use of `require`. Please don't mix `assert` with `require`.